amd64/bsd_asm.h

```diff
@@ -40,7 +40,7 @@
 #include "../i387/osx_asm.h"
 #define CNAME(x) EXT(x)
 #else
-#include "bsd_cdefs.h"
+#include <sys/cdefs.h>
```

Comment:
That's what I feared. I hate C. You can't rely on these macros to be defined as the header is private, but if you define them you get a redefinition warning.

Let's go back to only checking for `_SYS_CDEFS_H` and `_SYS_CDEFS_H`.